docs/en/development/CI.md

```diff
@@ -0,0 +1,209 @@
+
+#### Report details
+
+[Status page example](https://clickhouse-builds.s3.yandex.net/12550/67d716b5cc3987801996c31a67b31bf141bc3486/clickhouse_build_check/report.html).
+
+- **Compiler**: `gcc-9` or `clang-10` (or `clang-10-xx` for other architectures e.g. `clang-10-freebsd`).
+- **Build type**: `Debug` or `RelWithDebInfo` (cmake).
+- **Sanitizer**: `none` (without sanitizers), `address` (ASan), `memory` (MSan), `undefined` (UBSan), or `thread` (TSan).
+- **Bundled**: `bundled` build uses system libraries, and `unbundled` build uses libraries from `contrib` folder.
+- **Splitted** `splitted` is a [split build](build.md#split-build)
+- **Status**: `success` or `fail`
+- **Build log**: link to the building and files copying log, useful when build failed.
+- **Build time**.
+- **Artifacts**: build result files (with `XXX` being the server version e.g. `20.8.1.4344`).
+    - `clickhouse-client_XXX_all.deb`
+    - `clickhouse-common-static-dbg_XXX[+asan, +msan, +ubsan, +tsan]_amd64.deb`
+    - `clickhouse-common-staticXXX_amd64.deb`
+    - `clickhouse-server_XXX_all.deb`
+    - `clickhouse-test_XXX_all.deb`
+    - `clickhouse_XXX_amd64.buildinfo`
+    - `clickhouse_XXX_amd64.changes`
+    - `clickhouse`: Main built binary.
+    - `clickhouse-odbc-bridge`
+    - `unit_tests_dbms`: GoogleTest binary with ClickHouse unit tests.
+    - `shared_build.tgz`: build with shared libraries.
+    - `performance.tgz`: Special package for performance tests.
+
+### Special build check
+Performs static analysis and code style checks using `clang-tidy`. The report is similar to the [build check](#build-check). Fix the errors found in the build log.
+
+
+### Functional stateless tests
+Runs [stateless functional tests](tests.md#functional-tests) for ClickHouse
+binaries built in various configurations -- release, debug, with sanitizers,
+etc. Look at the report to see which tests fail, then reproduce the failure
+locally as described [here](tests.md#functional-test-locally). Note that you
+have to use the correct build configuration to reproduce -- a test might fail
+under AddressSanitizer but pass in Debug. Download the binary from [CI build
+checks page](build.md#you-dont-have-to-build-clickhouse), or build it locally.
+
+
+### Functional stateful tests
+Runs [stateful functional tests](tests.md#functional-tests). Treat them in the same way as the functional stateless tests. The difference is that they require `hits` and `visits` tables from the [Yandex.Metrica dataset](../getting-started/example-datasets/metrica.md) to run.
+
+
+### Integration tests
+Runs [integration tests](tests.md#integration-tests).
+
+
+### Testflows check
+Runs some tests using Testflows test system. See [here](https://github.com/ClickHouse/ClickHouse/tree/master/tests/testflows#running-tests-locally) how to run them locally.
```

Comment:
Additional/special?

Reply:
Not sure, these are normal tests, just use a different test system.